<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommInterface</class>
 <widget class="QWidget" name="CommInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>711</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>170</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>210</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>330</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>View Degree Info</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>330</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>View Plan/Add Courses</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Student Info&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>211</width>
       <height>261</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>261</height>
      </size>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Student Name</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Major</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Start Year</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Total Cred Hours</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Degree Cred Hours</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>GPA</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Data</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;Student Plan&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>320</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Student Info</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>320</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Degree Requirements</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>231</width>
       <height>281</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>229</width>
        <height>279</height>
       </rect>
      </property>
      <widget class="QScrollBar" name="verticalScrollBar">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>0</y>
         <width>21</width>
         <height>281</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>211</width>
         <height>281</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Course</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Remove Course</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>210</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Grade</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>130</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Semester Note</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;Degree Requirements&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>370</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>View Plan/Add Courses</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>370</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Student Info</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>291</width>
       <height>211</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>Foundations</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Core</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Gen Electives</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>EE Foundation</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Adv Comp</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Capstone</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Num Taken</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Num Left</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>6</pointsize>
       </font>
      </property>
     </item>
    </widget>
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>140</y>
       <width>291</width>
       <height>211</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>Western</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Non-West</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Libed</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Soc/Behav</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>US Min</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Hum/Arts</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Compos</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Num Taken</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Num Left</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Major Specific</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>100</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>General Requirements</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
